### ISSUE TITLE
Irrelevant warning while showing bears

### ISSUE THREAD

**Issue by Dawny33** [coala/coala#2775]
### Overview Description

When the user does `coala --show-bears --filter-by-language <language>`,  and if the directory does not have a `.coafile`, it returns the following errors:

```
[WARNING][16:10:15] The default coafile '.coafile' was not found. You can generate a configuration file with your current options by adding the `--save` flag.
[WARNING][16:10:15] coala will not run any analysis. Did you forget to give the `--files` argument?
[WARNING][16:10:15] coala will not run any analysis. Did you forget to give the `--bears` argument?
```

Out of these, the 2nd and the 3rd warnings are not required, as the user is trying to look at the list of bears and not trying to execute any analysis

In this, 
# Steps to Reproduce
1. Go to a directory which does not have a `.coafile`
2. Try to see the bears, filtering by a language, say Python, coala --show-bears --filter-by-language Python
# Actual Results

```
[WARNING][16:10:15] The default coafile '.coafile' was not found. You can generate a configuration file with your current options by adding the `--save` flag.
[WARNING][16:10:15] coala will not run any analysis. Did you forget to give the `--files` argument?
[WARNING][16:10:15] coala will not run any analysis. Did you forget to give the `--bears` argument?
```
# Expected Results

```
[WARNING][16:10:15] The default coafile '.coafile' was not found. You can generate a configuration file with your current options by adding the `--save` flag.
```

**Comment by Makman2** [comment]
Fpr for `--show-bears` no .coafile shall be necessary.

**Comment by zharkovav** [comment]
Please assign it to me)

**Comment by Makman2** [comment]
sure :)

**Comment by sils** [comment]
Unassigned @zharkovav as per request. Please don't ping me personally for such things. Everyone can do them - and almost everyone pings me. If you don't address a person specifically you'll likely get help faster from other people!

(Don't worry, this is not a problem but keeping things general is usually a good policy :))

**Comment by incorrectusername** [comment]
How to check if the changes i did are correct?

**Comment by Nosferatul** [comment]
enter in a venv and use pip3 install -e <path_to_your_local_repo>. More details -> http://api.coala.io/en/latest/Developers/Development_Setup.html

**Comment by incorrectusername** [comment]
I need some help regarding how to write test for this bug?i checked writing tests section from docs.coala.io. any suggestion. can you direct me how to do it.

**Comment by jayvdb** [comment]
Look at the other tests for this module.

**Comment by satwikkansal** [comment]
Fixed with https://github.com/coala/coala/pull/4622

**Comment by satwikkansal** [comment]
I combined few of desired changes for https://github.com/coala/coala/issues/4621 (very minor) along with the fix for this issue in #4622
